### DISCUSSION TITLE
Config no longer allows backpack recipes to be disabled

### DISCUSSION THREAD

**Discussion by thetitan555** [P3pp3rF1y/SophisticatedBackpacks#502] [Ideas]
In 1.16, the crafting recipes for all items could be disabled from the config. This is no longer possible in 1.18. Removing the recipe with CraftTweaker is possible, but the old recipe still shows up in JEI (can't be hidden with JEITweaker).

**Comment by LilMissMasq** [comment]
I can second this being an issue.

**Comment by userofbricks** [comment]
just use a datapack to set the .json file as empty.
